devito/ops/types.py

```diff
@@ -31,7 +31,7 @@ class OpsAccessible(basic.Symbol):
     """
     is_Scalar = True
 
-    def __new__(cls, name, dtype, read_only, *args, **kwargs):
+    def __new__(cls, name, dtype, read_only, * args, **kwargs):
```

Comment:
drop leftover

Reply:
Strange, in my local branch does not have this space.

Reply:
Forget what I said, I was looking in the wrong branch.

Fixed!

Reply:
no it's not fixed